resolver/rewriter_resolver.go

```diff
@@ -126,7 +126,7 @@ func (r *RewriterResolver) rewriteRequest(logger *logrus.Entry, request *dns.Msg
 			rewritten.Question[i].Name = dns.Fqdn(domainRewritten)
 
 			logger.WithFields(logrus.Fields{
-				"domain":  domainOriginal,
+				"domain":  util.Obfuscate(domainOriginal),
 				"rewrite": rewriteKey + ":" + r.cfg.Rewrite[rewriteKey],
```

Comment:
Should we obfuscate this too? Cause it pretty much contains part of the domain?

Reply:
I added additional obfuscation